nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Updater/SdkPackageUpdater.cs

```diff
@@ -20,132 +20,124 @@ internal static class SdkPackageUpdater
         string previousDependencyVersion,
         string newDependencyVersion,
         bool isTransitive,
-        Logger logger
-    )
+        Logger logger)
     {
         // SDK-style project, modify the XML directly
         logger.Log("  Running for SDK-style project");
-        var buildFiles = await MSBuildHelper.LoadBuildFiles(repoRootPath, projectPath);
-
-        var newDependencyNuGetVersion = NuGetVersion.Parse(newDependencyVersion);
 
-        // update all dependencies, including transitive
+        var buildFiles = await MSBuildHelper.LoadBuildFiles(repoRootPath, projectPath);
```

Comment:
The changes in this file are mostly just grouping logic into methods to make this more readable.